### ISSUE TITLE
Update bei Adapterstart möglich?

### ISSUE THREAD

**Issue by Waldmensch1** [sirjojo69/ioBroker.awattar#12]
Ich habe den Adapter gerade frisch installiert. Es wäre schön, wenn der Adapter sich einmalig sofort die aktuellen Daten von Awattar holen und die Datenpunkte anlegen würde. Ich muss jetzt bis zum cronjob warten, um zu sehen ob es funktioniert. Ich weiß, man könnte jetzt den Cron umstellen und nach dem Update wieder auf 15:00 einstellen. Ist aber recht aufwändig und unnötig.

**Comment by sirjojo69** [comment]
hi... ich würd zwar sagen, dass das eher ein feature request als ein issue ist aber ich nehme es in die nächste version mit rein... macht schon sinn.

**Comment by Waldmensch1** [comment]
Jo, ist es. Ich kann es aber nicht selbst als feature request taggen ;)

**Comment by Waldmensch1** [comment]
Moin, macht ihr am Adapter noch was? :)

**Comment by sirjojo69** [comment]
jo... nur sehr wenig zeit im moment. wollte eigentlich zwischen den jahren hier weiter machen, aber....

**Comment by sirjojo69** [comment]
feature ist jetzt in version 1.1.0 umgesetzt (bisher nur hier in github verfügbar; [ wers ausprobieren will muss "über die Katze installieren"]
